### PR TITLE
fix: issue-#1418 refetch following count after following a user

### LINKED ISSUE #1418: Feature: refetch following count after following a user
### Type of feature

🍕 Feature

### Current behavior

https://github.com/open-sauced/insights/assets/79809121/ba7cc627-33f0-476b-90fb-0b5cd0b7df27

### Suggested solution

Refetch the following count each time a new user is followed

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by aryanas159**
## Description

fixes #1418 
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by aryanas159** [comment]
https://github.com/open-sauced/app/assets/114330931/587ce912-afa3-44bd-b86a-c3ba456197a5

**Review by OgDev-01 — APPROVED** [review]
LGTM

File: components/atoms/TopContributorCard/top-contributor-card.tsx [hunk 1]
```diff
@@ -11,9 +11,10 @@ import Button from "../Button/button";
 
 export interface TopContributorCardProps {
   login: string;
+  mutateLoggedInUser: () => void;
```
Comment (brandonroberts) [inline components/atoms/TopContributorCard/top-contributor-card.tsx:14]: nit: This naming is pretty specific to SWR

```suggestion
  refreshLoggedInUser: () => void;
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: components/atoms/TopContributorCard/top-contributor-card.tsx (hunk 1 above)
Comment (aryanas159) [reply components/atoms/TopContributorCard/top-contributor-card.tsx:14]: @brandonroberts Okay, I'll change it

**Review by brandonroberts — APPROVED** [review]
♻️

**Pull request merged**